libs/platform/table/table.component.scss

```diff
@@ -492,6 +492,7 @@ fdk-dynamic-portal {
     &__body-table {
         &.fd-table--pop-in {
             .fd-table__row--secondary {
+                cursor: pointer;
```

Comment:
Made the secondary row clickable and added a pointer cursor to indicate interactivity.
 @mikerodonnell89 @InnaAtanasova

Reply:
I think it also needs the active/hover state as well and should be applied to the entire cell